src/Input/Input.js

```diff
@@ -70,7 +76,7 @@ export const styleSheet = createStyleSheet('MuiInput', (theme) => {
     },
     disabled: {
       color: theme.palette.text.disabled,
-      cursor: 'not-allowed',
+      cursor: 'not-allowed !important',
```

Comment:
It is... I have a commit that fixes this by using a separate enabled class for the select element that sets the cursor to pointer. There might be a better way but it seems to work well.